doc/changes/latest.inc

```diff
@@ -29,6 +29,8 @@ Enhancements
 
 Bugs
 ~~~~
+- Fix :func:`mne.io.read_raw_curry` to deal with Curry datasets that have channels that are listed in the labels file, but which are absent from the saved data file (e.g. 'Ref' channel).  Also now populates info['meas_date'] if possible, by **by new contributor** |Tod Flak|_ (:gh:`8400`)
```

Comment:
Looks like it's missing the `.. |Eduard Ort| replace:: **Eduard Ort**`-equivalent line above

Reply:
OK, wasn't sure that was necessary, so didn't do it ... will do now.

Reply:
Indeed `|whatever|` is RST [substitution syntax](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/reuse.html#use-a-substitution) so we have to give it the thing to substitute (and FYI the trailing `_` is link-syntax, which turns it into a link)

Reply:
Lots to learn.  Thanks for all your hand-holding.  I can't imagine how you have time for all your real work if dealing with people like me all the time!

Reply:
For me this is part of my work, you can thank NIH/NIMH (among others) :)

There is always a learning curve / startup cost, don't sweat it! @agramfort had to spend many hours teaching me years ago as well...